src/GDGUkraine/utils.py

```diff
@@ -114,3 +122,283 @@ def make_vcard(user_reg, url=None):
+
+                    if res.get(key_cls):
+                        continue
+
+                    res[key_cls] = {
+                        'args': params,
+                        'url': (uri
+                                if uri and uri.endswith('/')
+                                else '/'.join([uri]))}
+            elif not inspect.isfunction(hndlr) and \
+                    not isinstance(hndlr, property) and \
+                    not method.startswith('__'):
+                # Looks like we have another class instance mounted and nested
+                # import ipdb; ipdb.set_trace()
+                res.update(
+                    retrieve_class_routes(
+                        cls=hndlr,
+                        mp=(''
+                            if uri.endswith('/')
+                            else '/').join([uri, method]),
+                        handler_cls='.'.join([handler_cls, method])))
+        # TODO: handle `index` and `default` methods
+        if res.get(handler_cls):
+            res['.'.join([handler_cls, 'index'])] = res[handler_cls]
+        return res
+
+    global url_resolve_map
+    urls = {'__routes__': {}}
+    if url_resolve_map is None or force:
+        for script in cp.tree.apps:
+            app = cp.tree.apps[script]
+            if hasattr(cp.lib, 'gctools') and \
+                    isinstance(app.root, cp.lib.gctools.GCRoot):
+                logger.debug('It is CherryPy garbage collector app')
+                logger.debug('There are tests running probably')
+                logger.debug('Skipping...')
+                continue
+            request_dispatcher = app.config['/'].get('request.dispatch')
+            if app.root is not None:
+                logger.debug('It is class-based routed app')
+                logger.debug(script)
+                logger.debug(app)
+                urls.update(retrieve_class_routes(app.root,
+                                                  mp=app.script_name))
+            elif isinstance(request_dispatcher,
+                            cp._cpdispatch.RoutesDispatcher):
+                logger.debug('It is Routes routed app')
+                logger.debug('Skipping...')
+                logger.debug(script)
+                logger.debug(app)
+                logger.debug(request_dispatcher)
+                for handler_name in request_dispatcher.controllers.keys():
+                    if handler_name in urls['__routes__']:
+                        logger.warn('Handler name `{}` is already in routes '
+                                    'URL map. Avoid having same identifier for'
+                                    ' different paths!'.format(handler_name))
+                    urls['__routes__'][handler_name] = {
+                        'script': app.script_name,
+                        'mapper': request_dispatcher.mapper}
+        url_resolve_map = urls
+        return urls
+    else:
```

Comment:
You do not need lines 301 and 302
Just unindent line 303

So instead of 
```python
if condition:
   do_one()
   return result
else:
   return result
```

you will have
```python
if condition:
   do_one()
return result